src/cli/cli.cpp

```diff
@@ -2733,6 +2745,18 @@ template <> otError Interpreter::Process<Cmd("log")>(Arg aArgs[])
 #if OPENTHREAD_CONFIG_LOG_LEVEL_DYNAMIC_ENABLE
             uint8_t level;
 
+            /**
+             * @cli log level <level>
```

Comment:
should be 

`@cli log level (set)`

Parameters don't go in the CLI name, only on the cparam line